MerlinAU.sh

```diff
@@ -3229,17 +3225,19 @@ EOT
     cd "$FW_BIN_DIR"
 
     ##----------------------------------------##
-    ## Modified by Martinski W. [2024-Feb-10] ##
+    ## Modified by Martinski W. [2024-Mar-16] ##
     ##----------------------------------------##
     local checkChangeLogSetting="$(Get_Custom_Setting "CheckChangeLog")"
 
-    if [ "$checkChangeLogSetting" = "ENABLED" ]; then
-        # Files matching the pattern 'Changelog-*.txt' #
-        changelog_file="$(/usr/bin/find -L "${FW_BIN_DIR}" -name "Changelog-*.txt" -print | head -n 1)"
+    if [ "$checkChangeLogSetting" = "ENABLED" ]
+    then
+        # Get the correct Changelog filename (Changelog-[386|NG].txt) based on the "build number" #
+        changeLogTag="$(echo "$(nvram get buildno)" | grep -qE "^386[.]" && echo "386" || echo "NG")"
```

Comment:
Smart call again, approved; I was wondering if you were going to make it based on the build number or the model number. (Considering we only had 2 models still getting the 386 builds) but build number is consistent with what we have previously been using, model numbers are really only for model specific changes at this point (such as the blocks)